core/storage/exploration/gae_models.py

```diff
@@ -389,7 +391,9 @@ class ExpSummaryModel(base_models.BaseModel):
     owner_ids = ndb.StringProperty(indexed=True, repeated=True)
     # The user_ids of users who are allowed to edit this exploration.
     editor_ids = ndb.StringProperty(indexed=True, repeated=True)
-    # The user_ids of users who are allowed to translate this exploration.
+    # The user_ids of users who are allowed to voiceover this exploration.
+    voice_artist_ids = ndb.StringProperty(indexed=True, repeated=True)
+    # DEPRECATED in v2.8.1. Do not use.
```

Comment:
Ditto!

Reply:
Done. Should I submit the form for testing on the test server?

Reply:
Yes, please, if you want this run during the June release.